src/modules/new-request-form/fields/attachments/FileListItem.tsx

```diff
@@ -55,13 +55,26 @@ export function FileListItem({
 
   return (
     <FileList.Item>
-      <File type="generic" title={fileName} onKeyDown={handleFileKeyDown}>
+      <File
+        type="generic"
+        tabIndex={0}
+        aria-label={t(
+          "new-request-form.attachments.file",
+          "File: {{fileName}}, press delete to remove",
+          { fileName }
+        )}
+        onKeyDown={handleFileKeyDown}
+      >
         {file.status === "pending" ? (
           <>
             <FileNameWrapper>{fileName}</FileNameWrapper>
             <Tooltip content={stopUploadLabel}>
               <File.Close
-                aria-label={stopUploadLabel}
+                aria-label={t(
+                  "new-request-form.attachments.stop-upload-aria-label",
+                  "Stop uploading {{fileName}}",
+                  { fileName }
+                )}
```

Comment:
Can we please add `aria-describedby={undefined}` to avoid a duplicate screen reader announcement here, too?

```suggestion
                aria-label={t(
                  "new-request-form.attachments.stop-upload-aria-label",
                  "Stop uploading {{fileName}}",
                  { fileName }
                )}
                aria-describedby={undefined}
```

* Current screen reader announcement: _Stop uploading [file name], Stop uploading, button_
* Expected screen reader announcement: _Stop uploading [file name], button_

(The word "button" comes from it being an HTML button element.)

Reply:
Ok I applied suggestion